tools/presenter/vite.config.ts

```diff
@@ -13,7 +13,7 @@ import rehypeHighlight from 'rehype-highlight';
 import { defineConfig } from 'vite';
 import { VitePluginFonts } from 'vite-plugin-fonts';
 
-import { ConfigPlugin } from '@dxos/config/vite-plugin';
+import { ConfigPlugin } from '@dxos/config/dist/plugin/node/vite-plugin.cjs';
```

Comment:
Is this because of this being a module?

Reply:
Seems like it just isn’t exported anywhere, and AFAIK there isn’t an alias that would resolve to this, though maybe you know of one?

Reply:
Well, there is something there

https://github.com/dxos/dxos/blob/main/packages/sdk/config/vite-plugin.js
https://github.com/dxos/dxos/blob/main/packages/sdk/config/package.json#L21

it's just not a module